lib/traject/psulib_config.rb

```diff
@@ -26,9 +27,9 @@
   provide 'log.file', 'log/traject.log'
   provide 'log.error_file', 'log/traject_error.log'
   provide 'solr_writer.commit_on_close', 'true'
+  provide 'reader_class_name', 'Traject::MarcCombiningReader'
```

Comment:
So, `Traject::MarcCombiningReader` requires `jruby`, and it even calls `marc4j_reader` itself. I would guess then, that extractions would fail when using MRI. However, I'm testing now and the transformation process seems to be running just fine. So... 🤷‍♂️ 

I guess my point was that we might want to move this reader class in the is_jruby conditional block.

Also, the MarcCombiningReader class looks for `JRUBY_VERSION` -- we should define that, right?